### PR TITLE
Add Conductor to Java section

### PR DESCRIPTION
**Pull request by ddavison**
Conductor is a framework built with Selenium, that many companies now implement. (companies such as [Major League Gaming](http://majorleaguegaming.com), [Inbound Now](http://www.inboundnow.com/))

This framework in specific makes test writing very easy by implementing a nice fluent-interface to the Selenium WebDriver API.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -82,4 +86,4 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ### Blogs
 
 - [Official Selenium Blog](http://seleniumhq.wordpress.com/) - The official blog by SeleniumHQ.
-- [Elemental Selenium](http://elementalselenium.com/) - A free, once-weekly e-mail on how to use Selenium like a Pro.
\ No newline at end of file
+- [Elemental Selenium](http://elementalselenium.com/) - A free, once-weekly e-mail on how to use Selenium like a Pro.
```
Comment (ddavison) [inline readme.md:89]: if this is newline at EOF is a problem let me know :+1: 

the built-in github editor doesn't do a great job abiding by the contribution guidelines itself.

Comment (christian-bromann) [inline readme.md:89]: well I don't think it is that important for this matter :wink:

**Comment by christian-bromann** [comment]
Yeah!! First contribution on this "project". Thanks a lot!

**Comment by ddavison** [comment]
cool! :D

**Pull request merged**